chpl/chpl-service/src/main/java/gov/healthit/chpl/realworldtesting/domain/RealWorldTestingUpload.java

```diff
@@ -0,0 +1,19 @@
+package gov.healthit.chpl.realworldtesting.domain;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class RealWorldTestingUpload implements Serializable {
+    private static final long serialVersionUID = -5933226973828290819L;
+
+    private String chplProductNumber;
+    private LocalDate lastChecked;
+    private RealWorldTestingType  type;
```

Comment:
```suggestion
    private RealWorldTestingType type;
```

I know it's petty, and will not hold up anything